### DISCUSSION TITLE
How to pick viewbox/width/height

### DISCUSSION THREAD

**Discussion by Salosy** [Yqnn/svg-path-editor#107] [General]
I'm using svg-path-editor and text editor exclusively to create svgs. How should I go about picking viewbox? I'm worried that I will create too many icons that I will have to refactor later because my viewbox is too big or too small. They all squire shape if that matters.

Right now I use 200 200 viewbox and always have this path: `M0 0 200 0 200 200 0 200Z` inside svg-path-editor to be aware of boundaries.

**Comment by kuubeu** [comment]
There's no right or wrong way of picking a viewBox size, if you have trouble you can look at what other icon packs use. If you want pixel-perfect UI icons, take a look at [Google's Material Icons](https://fonts.google.com/icons?icon.style=Filled&icon.set=Material+Icons) and their [guidelines](https://m2.material.io/design/iconography/system-icons.html#grid-and-keyline-shapes). App icons are usually multiples of 2, like 64, 128, 256 and 512 (see [SuperTinyIcons](https://github.com/edent/SuperTinyIcons/) as an example), whereas fonts and icon fonts are usually 1000×1000.

As a side note, you can use `M0 0H200V200H0Z` to draw the boundaries, where `H` is a horizontal line and `V` is a vertical line — saves some characters :)
